### PR TITLE
Remove obsolete

### PR DESCRIPTION
**Pull request by domagojmedo**
# MOVED TO https://github.com/xamarin/Xamarin.Forms/pull/13813

### Description of Change ###

Removed obsolete members

### Issues Resolved ### 
https://github.com/dotnet/maui/issues/21

### API Changes ###

Removed obsolete members from the task except 

1. GetSizeRequest from Layout and VisualElement (https://github.com/dotnet/maui/issues/21#issuecomment-708660997)

1. Platform.iOS.WebViewRenderer, I can't find it, I guess it's already gone

1. Platform.Android.AndroidActivity, I can't find it

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Review by PureWeen — CHANGES_REQUESTED** [review]
```
"D:\agent\1\s\Xamarin.Forms.sln" (Build target) (1) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF.csproj" (default target) (41) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF.csproj" (Build target) (41:3) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_5eqf5jgc_wpftmp.csproj" (_CompileTemporaryAssembly target) (43) ->
  Platform.cs(14,5): error CS0246: The type or namespace name 'IPlatform' could not be found (are you missing a using directive or an assembly reference?) [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_5eqf5jgc_wpftmp.csproj]
  Platform.cs(270,15): error CS0246: The type or namespace name 'IPlatform' could not be found (are you missing a using directive or an assembly reference?) [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_5eqf5jgc_wpftmp.csproj]
  Platform.cs(270,15): error CS0538: 'IPlatform' in explicit interface declaration is not an interface [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_5eqf5jgc_wpftmp.csproj]

    49 Warning(s)
    154 Error(s)
```

**Comment by domagojmedo** [comment]
Unfortunately I have problems building the solution (even after deleting bin/obj folder) so I probably missed some of these errors.

**Comment by PureWeen** [comment]
```
"D:\agent\1\s\Xamarin.Forms.sln" (Build target) (1) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF.csproj" (default target) (41) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF.csproj" (Build target) (41:3) ->
"D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_atbgcpsw_wpftmp.csproj" (_CompileTemporaryAssembly target) (43) ->
  Extensions\FormattedStringExtensions.cs(30,24): error CS1061: 'Span' does not contain a definition for 'Font' and no accessible extension method 'Font' accepting a first argument of type 'Span' could be found (are you missing a using directive or an assembly reference?) [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_atbgcpsw_wpftmp.csproj]
  Renderers\LabelRenderer.cs(65,37): error CS0117: 'Label' does not contain a definition for 'FontProperty' [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_atbgcpsw_wpftmp.csproj]
  Renderers\LabelRenderer.cs(151,91): error CS1061: 'Label' does not contain a definition for 'Font' and no accessible extension method 'Font' accepting a first argument of type 'Label' could be found (are you missing a using directive or an assembly reference?) [D:\agent\1\s\Xamarin.Forms.Platform.WPF\Xamarin.Forms.Platform.WPF_atbgcpsw_wpftmp.csproj]

```

**Comment by PureWeen** [comment]
```
"D:\agent\1\s\Xamarin.Forms.Platform.Tizen\Xamarin.Forms.Platform.Tizen.csproj" (default target) (38:2) ->
(CoreCompile target) -> 
  FormsApplication.cs(194,18): error CS1061: 'Application' does not contain a definition for 'Platform' and no accessible extension method 'Platform' accepting a first argument of type 'Application' could be found (are you missing a using directive or an assembly reference?) [D:\agent\1\s\Xamarin.Forms.Platform.Tizen\Xamarin.Forms.Platform.Tizen.csproj]
  Renderers\LabelRenderer.cs(17,34): error CS0117: 'Label' does not contain a definition for 'FontProperty' [D:\agent\1\s\Xamarin.Forms.Platform.Tizen\Xamarin.Forms.Platform.Tizen.csproj]
```

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]
@domagojmedo  There are some conflicts, could you review them?

**Comment by domagojmedo** [comment]
@jsuarezruiz conflicts are resolved

File: src/Platform.Renderers/src/Xamarin.Forms.Platform.Android/Renderers/SliderRenderer.cs [hunk 1]
```diff
@@ -24,34 +23,23 @@ public SliderRenderer(Context context) : base(context)
...
-		}
-
-		[PortHandler]
 		double Value
 		{
 			get { return _min + (_max - _min) * (Control.Progress / 1000.0); }
 			set { Control.Progress = (int)((value - _min) / (_max - _min) * 1000.0); }
 		}
 
-		[PortHandler]
```
Comment (PureWeen) [inline src/Platform.Renderers/src/Xamarin.Forms.Platform.Android/Renderers/SliderRenderer.cs:41]: Keep these here

File: src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs [hunk 2]
```diff
@@ -64,10 +66,6 @@ public class LabelRenderer : ViewRenderer<Label, TextBlock>
 		//	return new TextBlockAutomationPeer(Control);
 		//}
 
-		public LabelRenderer()
```
Comment (PureWeen) [inline src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs:67]: Did we want to delete this?

File: src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs [hunk 3]
```diff
@@ -267,11 +265,23 @@ void UpdateFont(TextBlock textBlock)
 			if (label == null || (label.IsDefault() && !_fontApplied))
 				return;
 
-#pragma warning disable 618
-			Font fontToApply = label.IsDefault() && _isInitiallyDefault ? Font.SystemFontOfSize(NamedSize.Medium) : label.Font;
-#pragma warning restore 618
+			if (label.IsDefault() && _isInitiallyDefault)
+			{
+				var font = Font.SystemFontOfSize(NamedSize.Medium);
```
Comment (PureWeen) [inline src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs:270]: Where is this code coming from?

Comment (mattleibow) [reply src/Platform.Renderers/src/Xamarin.Forms.Platform.UAP/LabelRenderer.cs:270]: That was in the previous PR... Not sure what it should be

File: src/Platform.Renderers/src/Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs [hunk 4]
```diff
@@ -83,28 +83,23 @@ public ShellSectionRenderer(IShellContext context, Type navigationBarType, Type
 			_context.Shell.PropertyChanged += HandleShellPropertyChanged;
 		}
 
-		[Export("navigationBar:shouldPopItem:")]
```
Comment (PureWeen) [inline src/Platform.Renderers/src/Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs:86]: Did we want to delete this?

File: src/Platform.Renderers/src/Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs [hunk 5]
```diff
@@ -528,9 +523,6 @@ async void SendPoppedOnCompletion(Task popTask)
 
 			var poppedPage = _shellSection.Stack[_shellSection.Stack.Count - 1];
 
-			// this is used to setup appearance changes based on the incoming page
-			((IShellSectionController)_shellSection).SendPopping(popTask);
```
Comment (PureWeen) [inline src/Platform.Renderers/src/Xamarin.Forms.Platform.iOS/Renderers/ShellSectionRenderer.cs:532]: I dont think we want to delete this

**Review by PureWeen — CHANGES_REQUESTED** [review]
Left a few comments

**Comment by mattleibow** [comment]
Moving to https://github.com/xamarin/Xamarin.Forms/pull/13813 for better CI reports

**Pull request closed without merging**